### DISCUSSION TITLE
咕FR：b站动态的关键字推送

### DISCUSSION THREAD

**Discussion by A2Sumie** [Tsuk1ko/CQ-picfinder-robot#474] [我有想法]
首先感谢把issues全清了stO

太边角的需求不好意思发issues里了xD

需求：检测到一组预设关键字后，超越video:true等把动态推出去
允许关键字里带斜杠如“22/7”
（关注22/7谢谢喵，虽然可爱妖精也毕业了）

+α：允许我在某处把这些关键字存成表然后按表名调关键字
转发动态只检测新动态里有没有关键字，且可选开关

cqps已经被我玩成除了picsearch啥都干了xD

**Comment by Tsuk1ko** [comment]
![](https://github.com/user-attachments/assets/2278640a-d2ab-4c6f-a00c-24fda8ecce7e) 要不还是咕了吧（

**Reply by A2Sumie** [reply]
要不还是咕了吧
